rpa-manager/modules/ROOT/pages/settings-adminsettings-reassignassets.adoc

```diff
@@ -19,7 +21,7 @@ The following assets are deleted on the deletion of the user:
 * Private dashboards
 * User API Keys
 
-User Tasks are not shown in the *Reassign Assets* view and can be reassigned via the *Unprocessed User Tasks* view.
+User Tasks are not shown in the *Reassign Assets* view and can be reassigned via the *Unprocessed User Tasks* view or via the run configurations of a process.
```

Comment:
Is "run configuration" a button or a shortcut? If so, it should go between asterisks, right? *Run Configuration*

Reply:
I don't think it is necessary to add the links to the RPA assets in the See Also section. 
If "Run Configurations" is an asset, it's enough to bold it so that the reader understands that it is an object.
I don't know if I'm clear enough :/ 
Thanks!